cmd/create/cluster/cmd_test.go

```diff
@@ -26,34 +27,34 @@ var _ = Describe("Validates OCP version", func() {
 	var _ = Context("when creating a hosted cluster", func() {
 
 		It("OK: Validates successfully a cluster for hosted clusters with a supported version", func() {
-			v, err := client.ValidateVersion("4.12.0", []string{"4.12.0"}, stable, false, true)
+			v, err := client.ValidateVersion("4.12.5", []string{"4.12.5"}, stable, false, true)
 			Expect(err).NotTo(HaveOccurred())
-			Expect(v).To(Equal("openshift-v4.12.0"))
+			Expect(v).To(Equal("openshift-v4.12.5"))
 		})
 
 		It("OK: Validates successfully a nightly version of OCP for hosted clusters "+
 			"with a supported version", func() {
-			v, err := client.ValidateVersion("4.12.0-0.nightly-2022-11-25-185455-nightly",
-				[]string{"4.12.0-0.nightly-2022-11-25-185455-nightly"}, nightly, false, true)
+			v, err := client.ValidateVersion("4.12.0-0.nightly-2023-04-10-222146",
+				[]string{"4.12.0-0.nightly-2023-04-10-222146"}, nightly, false, true)
 			Expect(err).NotTo(HaveOccurred())
-			Expect(v).To(Equal("openshift-v4.12.0-0.nightly-2022-11-25-185455-nightly-nightly"))
+			Expect(v).To(Equal("openshift-v4.12.0-0.nightly-2023-04-10-222146-nightly"))
 		})
 
 		It("KO: Fails with a nightly version of OCP for hosted clusters "+
```

Comment:
```suggestion
		It("OK: Fails with a nightly version of OCP for hosted clusters "+
```

Reply:
:boxing_glove:

Reply:
Wait are the "KO"s intentional because these tests are simulating a failure?

Reply:
@renan-campos Exactly, not sure who set it up haha